tests/everest/test_fm_plugins.py

```diff
@@ -68,7 +67,7 @@ def get_forward_models(self):
 
     pm = plugin_manager(Plugin1(), Plugin2())
 
-    jobs = list(chain.from_iterable(pm.hook.get_forward_models()))
+    jobs = set(chain.from_iterable(pm.hook.get_forward_models()))
```

Comment:
Why `set` instead of `list` here? Did you get duplicate forward models showing up